packages/editor/src/lib/editor/Editor.ts

```diff
@@ -3750,7 +3750,7 @@ export class Editor extends EventEmitter<TLEventMap> {
 	 * @public
 	 */
 	@computed getPages(): TLPage[] {
-		return this._getAllPagesQuery().get().sort(sortByIndex)
```

Comment:
I think the `records` method creates new arrays each time, so this might not be needed

Reply:
this is required because the records method returns a computed value and it's not safe to mutate it with the .sort method becuase then the new shallow equality checks fail.